### PR TITLE
React Redux Capstone Project

### PR DESCRIPTION
**Pull request by Ogaga01**
## Hello Reviewer 👋 

**In this project milestone, I implemented the following requirements:**
- Add all dependencies required for the project
- Fetch data from the API
- Add data to the store
- Display data on the Home Page
- Display the Holiday details page
- Styled all components
- Add tests for Home Page, Country page, and Footer component
- Modify the Readme page
- Add presentation videos

Link to presentation video: https://www.loom.com/share/5e05ab6ea3ae45f1b932142e34fbced8

### CODE REVIEW

**Review by thecodechaser — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️ 

## Hi @Ogaga01👋, 

### You are doing a marvelous job on the React capstone project and this is one of the best projects I have seen so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight 🎉
- No linter errors. :heavy_check_mark:
- Gitflow is followed. :heavy_check_mark:
- Professional README file. :heavy_check_mark:
- PR has a good title and summary. ✔️
- All tests are passed. ✔️



## Required Changes ♻️
I can not highlight the code because this PR doesn't have that code as changed. I will request the changes here.
- [ ] On the homepage please also display some numeric data along with all the country's names as this is mentioned in the requirements. (You can display total holidays for all countries).

### See the image of the requirements

image

- [ ] For search functionalities if you search a country with all capital letters or without the first capital letter, It will not work. Please fix this issue so users can search without any letters case condition.
  - You can consider this code for your search filter. `const list = country.filter((e) => e.countryName.toLowerCase().includes(countries.toLowerCase()));`


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/Components/CountryCard.js [hunk 1]
```diff
@@ -20,7 +20,7 @@ const CountryTiles = () => {
 
   const searchHandler = (n) => {
     const countries = n.target.value;
-    const list = country.filter((e) => e.countryName.includes(countries));
+    const list = country.filter((e) => e.countryName.includes(countries.toLowerCase()));
```
Comment (danmainah) [inline src/Components/CountryCard.js]: - [ ] Kindly check on your search method, currently, I can't search a nation by typing the country's full name but when I delete the first letter of a nation I can search a country below is a picture demonstration:

Writing Full Name:
image

Writing All Letters Except the First Letter
image

**Review by danmainah — CHANGES_REQUESTED** [review]
Hi @Ogaga01 , 

Great work on making the changes requested by a previous reviewer 👏  👏 👏
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights
✔️ No linter errors
✔️ Github flow was correctly used
✔️  Good design


 # Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by julie-ify — APPROVED** [review]
Hi @Ogaga01,

✔️ Good design
✔️ Search functionality

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉

# Status: Approved ✔️

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**